samples/snippets/src/test/java/com/google/datastore/snippets/ConceptsTest.java

```diff
@@ -1044,4 +1044,81 @@ public void testPropertyFilteringRunQuery() {
         ImmutableMap.of("Task", ImmutableSet.of("priority", "tag"));
     assertEquals(expected, propertiesByKind);
   }
+
+  @Test
+  public void testInQuery() {
+    setUpQueryTests();
+    // [START datastore_in_query]
+    Query<Entity> query =
+        Query.newEntityQueryBuilder()
+            .setKind("Task")
+            .setFilter(
+                PropertyFilter.in("tag", ListValue.of("learn", "study"))
+            )
+            .build();
+    // [END datastore_in_query]
+    assertValidQuery(query);
+  }
+
+  @Test
+  public void testNotEqualsQuery() {
+    setUpQueryTests();
+    // [START datastore_not_equals_query]
+    Query<Entity> query =
+        Query.newEntityQueryBuilder()
+            .setKind("Task")
+            .setFilter(
+                PropertyFilter.neq("category", "Work")
+            )
+            .build();
+    // [END datastore_not_equals_query]
+    assertValidQuery(query);
+  }
+
+  @Test
+  public void testNotInQuery() {
+    setUpQueryTests();
+    // [START datastore_not_in_query]
+    Query<Entity> query =
+        Query.newEntityQueryBuilder()
+            .setKind("Task")
+            .setFilter(
+                PropertyFilter.not_in("category", ListValue.of("Work", "Chores", "School"))
+            )
+            .build();
+    // [END datastore_not_in_query]
+    assertValidQuery(query);
+  }
+
+  @Test
+  public void testEqQuerySorted() {
+    setUpQueryTests();
+    // [START datastore_eq_query_sorted]
+    Query<Entity> query =
+        Query.newEntityQueryBuilder()
+            .setKind("Task")
+            .setFilter(
+                PropertyFilter.eq("tag", "learn")
+            )
+            .setOrderBy(OrderBy.asc("tag"))
```

Comment:
to confirm, should we also be adding a not in sorted example?

Reply:
Thank you. This was intentional. Those two snippets explain a difference between == and IN. Here's the context from the docs:

Queries that include an equality filter for a given property ignore any sort order specified for that property. For example, the sort order is ignored in the following query:

datastore_eq_query_sorted snippet

This does not apply to queries that include an IN filter. Use the IN operator to combine up to 10 equality (==) clauses on the same property with a logical OR. If you add a sort order for that property, it's applied to the result set.

 datastore_in_query_sorted snippet